### PR TITLE
[Datasets] Add Mapillary Vistas Datasets to MMSeg Core Package.

### PR DESCRIPTION
**Pull request by AI-Tianlong**
## Motivation
Add Mapillary Vistas Datasets to core package.
Old PR #2484 

## Modification
- Add Mapillary Vistas Datasets to core package.
- Delete `tools/datasets_convert/mapillary.py` , dataset does't  need converting.
- Add `schedule_240k.py`  config.
- Add configs files.  
  ```none
  deeplabv3plus_r101-d8_4xb2-240k_mapillay_v1-512x1024.py
  deeplabv3plus_r101-d8_4xb2-240k_mapillay_v2-512x1024.py
  maskformer_swin-s_4xb2-240k_mapillary_v1-512x1024.py
  maskformer_swin-s_4xb2-240k_mapillary_v2-512x1024.py
  maskformer_r101-d8_4xb2-240k_mapillary_v1-512x1024.py
  maskformer_r101-d8_4xb2-240k_mapillary_v2-512x1024.py
  pspnet_r101-d8_4xb2-240k_mapillay_v1-512x1024.py
  pspnet_r101-d8_4xb2-240k_mapillay_v2-512x1024.py
  ```
- Synchronized changes to `projects/mapillary_datasets`

### CODE REVIEW

File: docs/zh_cn/user_guides/2_dataset_prepare.md [hunk 1]
```diff
@@ -451,3 +451,291 @@ python tools/convert_datasets/refuge.py --raw_data_root=/path/to/refuge/REFUGE2/
 ```
 
 其中包括 400 张图片的训练集, 400 张图片的验证集和 400 张图片的测试集.
+
+## Mapillary Vistas Datasets
+
+- Mapillary Vistas Datasets 可以在[此处](https://www.mapillary.com/dataset/vistas)注册后下载.
```
Comment (MeowZheng) [inline docs/zh_cn/user_guides/2_dataset_prepare.md:457]:
```suggestion
- Mapillary Vistas 数据集需要在[官方](https://www.mapillary.com/dataset/vistas)注册后下载.
```

File: mmseg/datasets/mapillary.py [hunk 2]
```diff
@@ -1,6 +1,68 @@
...
+                 [107, 142, 35], [0, 170, 30], [255, 255, 128], [250, 0, 30],
+                 [100, 140, 180], [220, 220, 220], [220, 128, 128],
+                 [222, 40, 40], [100, 170, 30], [40, 40, 40], [33, 33, 33],
+                 [100, 128, 160], [142, 0, 0], [70, 100, 150], [210, 170, 100],
+                 [153, 153, 153], [128, 128, 128], [0, 0, 80], [250, 170, 30],
+                 [192, 192, 192], [220, 220, 0], [140, 140, 20], [119, 11, 32],
+                 [150, 0, 255], [0, 60, 100], [0, 0, 142], [0, 0, 90],
+                 [0, 0, 230], [0, 80, 100], [128, 64, 64], [0, 0, 110],
+                 [0, 0, 70], [0, 0, 192], [32, 32, 32], [120, 10,
+                                                         10], [0, 0, 0]])
```
Comment (MeowZheng) [inline mmseg/datasets/mapillary.py:58]: how to get the meta information when use mapillary v2.0

**Review by MeowZheng — COMMENTED** [review]
I think if 240000 iters with SGD and 0.01 lr is a common setting, it should be add in `_base_`.

Moreover, please do not forget to add results in README and metafile.

File: mmseg/datasets/mapillary.py (hunk 2 above)
Comment (AI-Tianlong) [reply mmseg/datasets/mapillary.py:58]: There are two classes named `MapillaryDataset_v2_0(BaseSegDataset)` and `MapillaryDataset_v1_2(BaseSegDataset)` in this python file.
Both of them are imported in `__init__.py`https://github.com/open-mmlab/mmsegmentation/blob/a82662fd9ccd23c86493164b95d9d40f6ea70951/mmseg/datasets/__init__.py#L17
Datasets version can choose in configs following:
https://github.com/open-mmlab/mmsegmentation/blob/a82662fd9ccd23c86493164b95d9d40f6ea70951/configs/_base_/datasets/mapillary_v1_2.py#L2
https://github.com/open-mmlab/mmsegmentation/blob/a82662fd9ccd23c86493164b95d9d40f6ea70951/configs/_base_/datasets/mapillary_v2_0.py#L2
I will clearly write this in [docs](https://github.com/AI-Tianlong/mmsegmentation/blob/AI-Tianlong/add-mapillary-to-core-package/docs/en/user_guides/2_dataset_prepare.md)

**Comment by serser** [comment]
It seems blocked, CI Started 11d 15h 10m 46s ago?

**Review by xiexinch — APPROVED** [review]
We would continue to add other models.

**Pull request merged**